### PR TITLE
Heatmap: hover legend regardless of tooltip state

### PR DESCRIPTION
**Pull request by ryantxu**
This updates the new heatmap so it will always send hover events even if the tooltip is not visible.  This will let us show info in the legend without opening the tooltip

### CODE REVIEW

File: public/app/plugins/panel/heatmap-new/HeatmapHoverView.tsx [hunk 1]
```diff
@@ -170,7 +170,9 @@ export const HeatmapHoverView = ({ data, hover, showHistogram }: Props) => {
         <div>
           Bucket: {yDisp(yBucketMin)} - {yDisp(yBucketMax)}
         </div>
-        <div>Count: {count}</div>
+        <div>
+          {getFieldDisplayName(countField!, data.heatmap)}: {count}
```
Comment (leeoniya) [inline public/app/plugins/panel/heatmap-new/HeatmapHoverView.tsx:174]: can we memo this?

Comment (ryantxu) [reply public/app/plugins/panel/heatmap-new/HeatmapHoverView.tsx:174]: Maybe... But also not expensive -- the name is pulled from the cache on fiels

**Review by leeoniya — APPROVED** [review]
LGTM

File: public/app/plugins/panel/heatmap-new/HeatmapHoverView.tsx (hunk 1 above)
Comment (bobrik) [reply public/app/plugins/panel/heatmap-new/HeatmapHoverView.tsx:174]: What is the meaning of `1e-06` here? 

image

**Pull request merged**